### PR TITLE
Thankyou page integration and error block of contact form

### PR DESCRIPTION
**Pull request by Surya-Jamalpur**

### CODE REVIEW

File: bp-web-client/angular-app/src/app/components/app-contact/app-contact.component.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ import { Router } from "@angular/router";
 export class ContactComponent
   implements IDeactivateComponent, OnInit, OnDestroy
 {
+  isError: boolean = false;
```
Comment (pouncilt) [inline bp-web-client/angular-app/src/app/components/app-contact/app-contact.component.ts:30]: Add the appropriate variable scope here.  I.E (public, private, and or protected)

File: bp-web-client/angular-app/src/app/services/scroll-to.service.ts [hunk 2]
```diff
@@ -1,13 +1,20 @@
-import { Injectable } from '@angular/core';
-
+import { Injectable } from "@angular/core";
+import { BehaviorSubject } from "rxjs";
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: "root",
```
Comment (pouncilt) [inline bp-web-client/angular-app/src/app/services/scroll-to.service.ts:5]: Use single quotes instead of double quotes.

**Review by pouncilt — CHANGES_REQUESTED** [review]
Add the appropriate variable scope here. I.E (public, private, and or protected).
Also use single quotes instead of double quotes.

File: bp-web-client/angular-app/src/app/components/app-contact/app-contact.component.ts (hunk 1 above)
Comment (Surya-Jamalpur) [reply bp-web-client/angular-app/src/app/components/app-contact/app-contact.component.ts]: Added Public Scoping

File: bp-web-client/angular-app/src/app/services/scroll-to.service.ts (hunk 2 above)
Comment (Surya-Jamalpur) [reply bp-web-client/angular-app/src/app/services/scroll-to.service.ts:5]: changed to single quotes

**Pull request closed without merging**